02/2_header/header.h

```diff
@@ -0,0 +1,19 @@
+#ifndef HEADER_H
+#define HEADER_H
+
+int g_int_value = 0;

```

Comment:
move this to .c file. this is not a right example for global variable...

this also doesn't prevent multiple global variable definitions in exe file